conda_smithy/configure_feedstock.py

```diff
@@ -101,30 +101,39 @@ def render_circle(jinja_env, forge_config, forge_dir):
 
 
 @contextmanager
-def fudge_subdir(subdir):
+def fudge_subdir(subdir, meta=None):
```

Comment:
I'm going to update this so that we pass through the config, rather than the meta. Is that OK with you @msarahan?